CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## 0.14.0-rc1 / 2015-05-21
```

Comment:
@grobie So far, RC releases have been kind of under the radar, for internal early canarying. This time we actually posted an RC to the mailing list for the first time, so it might actually make sense to update the changelog earlier, yeah. I'd be fine with adding it to the Changelog, but then updating the existing entry upon the full release instead of adding yet another one.